src/VisualStudio/Core/Def/TableDataSource/Suppression/VisualStudioSuppressionFixService.cs

```diff
@@ -254,6 +254,8 @@ private async Task ApplySuppressionFixAsync(IEnumerable<DiagnosticData>? diagnos
             try
             {
                 using var token = _listener.BeginAsyncOperation(nameof(ApplySuppressionFix));
+
+                var originalSolution = _workspace.CurrentSolution;
```

Comment:
i haven't 100% confirmed that this is correct.

Reply:
these are some existing funky cases where we have things like command handlers that directly call into the underlying edit system.  i've skimmed and believe they're producing their actions/operations based on .CurrentSOlution, but i haven't deeply audited every single one of these cases yet.